app/assets/stylesheets/govuk-component/govspeak/_information-callout.scss

```diff
@@ -0,0 +1,22 @@
+// Govspeak information callout
+// http://govuk-component-guide.herokuapp.com/components/govspeak/fixtures/information_callout
+//
+// Support:
+//
+// - alphagov/whitehall: ✔︎
+// - alphagov/govspeak: ✔︎
+
+.govuk-govspeak {
+  .info-notice {
+    border-left: 1em solid $panel-colour;
+    padding: 1em 0 1em 1em;
+    margin: 2em 0;
+
+    // Remove margin from the last element
+    p:last-child,
```

Comment:
@36degrees When I use comments to explain something it usually makes sense to refactor into a named function:

eg `// Remove margin from the last element`
becomes `@include remove-last-element-margin;`

I prefer the selector to be qualified. It's faster and less risky (eg otherwise it would apply to all of the last list items in a list).

If we use a mixin it's easy to add another case if found.